src/client/utils/url.js

```diff
@@ -118,3 +120,10 @@ export function parseResourceType (resourceType) {
 export function stringifyResourceType (resourceType) {
     return sharedUrlUtils.getResourceTypeString(resourceType);
 }
+
+export function isChangedOnlyHash (currentUrl, newUrl) {
+    // NOTE: we compare proxied urls because urls passed into the function may be not proxied or proxied
+    // or relatives but the getProxyUrl function solves all problems related with it
```

Comment:
```
    // NOTE: we compare proxied urls because urls passed into the function may be proxied, non-proxied
    // or relative. The getProxyUrl function solves all the corresponding problems.
```